src/api/posts/test/feeds.test.js

```diff
@@ -75,16 +75,13 @@ describe('Test GET /feeds/delayed endpoint', () => {
 describe('GET /feeds/info', () => {
   test('Should return 200 and valid response object', async () => {
     function checkKeys(resBody) {
-      let bool = true;
       const allKeys = ['waiting', 'active', 'completed', 'failed', 'delayed', 'paused', 'jobCnt'];
-      const resKeys = Object.keys(resBody.queueInfo);
-      for (let i = 0; i < resKeys.length; i += 1) {
-        if (allKeys.indexOf(resKeys[i]) < 0 || typeof resBody.queueInfo[resKeys[i]] !== 'number') {
-          bool = false;
-          break;
+      return Object.keys(resBody.queueInfo).every((key) => {
```

Comment:
Can you leave a comment above this to explain what you're looking for here?  This code is a bit dense to read quickly.